### DISCUSSION TITLE
Checking the handling of specified software dependencies with m2e 1.20.1

### DISCUSSION THREAD

**Discussion by Markus-Elfring** [eclipse-m2e/m2e-core#880] [Q&A]
I am trying also to develop something with selected software components which are accordingly stored in the central Maven repository.
Thus I am using a development tool like the following.

Eclipse IDE for Java Developers 2022-06 (4.24.0)
Maven Integration for Eclipse Core 1.20.1.20220227-1319

I can put dependency specifcations for the components I became interested in then into the file “pom.xml” for another test project.
:eyes: I observe then that import specifcations can not be resolved as expected in the ways they were documented.
:thought_balloon: Are my observations similar to [information from the clarification request “Another case where m2e fails to import a Maven project correctly”](https://github.com/eclipse-m2e/m2e-core/discussions/859 "Clarification approach from 2022-07-21")?

I would appreciate further advices and solution ideas. :thinking:

**Comment by HannesWell** [comment]
Can you please share a minimal reproducer of your problem? I.e the Maven project pom.xml and corresponding java classes?
